vit-servicing-station-tests/src/common/data/generator/voting/template/arbitrary.rs

```diff
@@ -79,7 +79,7 @@ impl ArbitraryValidVotingTemplateGenerator {
             CompanyName()
                 .fake::<String>()
                 .to_lowercase()
-                .replace(" ", "-"),
+                .replace(' ', "-"),
```

Comment:
Interesting that both work, I think this is the first time I see this being used